stackit/internal/services/ske/cluster/resource.go

```diff
@@ -1337,11 +1337,13 @@ func latestMatchingVersion(availableVersions []ske.KubernetesVersion, providedVe
 	}
 
 	var versionUsed *string
+	var availableVersionsArray []string
```

Comment:
Maybe rename to something more descriptive, like `validAvailableVersions`?

`availableVersionsArray` doesn't tell more than the already existing array `availableVersions`

Reply:
It's literally just a string array of the `availableVersions`. We just need it to print in the error message. 
`availableVersions` includes more than just the `version`, but I don't think it's necessary to go into those details here